frontend/src/metabase/static-viz/components/PieChart/setttings.ts

```diff
@@ -0,0 +1,74 @@
+import type {
+  ComputedVisualizationSettings,
+  RemappingHydratedDatasetColumn,
+} from "metabase/visualizations/types";
+import type {
+  DatasetColumn,
+  PieChartSettings,
+  RawSeries,
+  VisualizationSettings,
+} from "metabase-types/api";
+import { getDefaultDimensionAndMetric } from "metabase/visualizations/lib/utils";
+import { getColorsForValues } from "metabase/lib/colors/charts";
+import { normalize } from "metabase-lib/queries/utils/normalize";
+import { getColumnKey } from "metabase-lib/queries/utils/get-column-key";
+
+import { SLICE_THRESHOLD } from "./constants";
+
+// TODO move this to base branch in shared location
```

Comment:
@alxnddr This function was in your old pie chart branch, and it seems like you added it to the combo chart branch. Can you move it down to the base echarts branch so we can share it?

Reply:
I updated the base branch with `getCommonStaticVizSettings`